bin/supervisor/src/flags/supervisor.rs

```diff
@@ -67,6 +78,64 @@ impl SupervisorArgs {
         })?;
         Ok(dependency_set)
     }
+
+    /// initialise and return the rollup config set.
+    pub async fn init_rollup_config_set(&self) -> Result<RollupConfigSet> {
+        let pattern = self.rollup_config_paths.to_string_lossy();
```

Comment:
probably to do struct decoding here, and rather throw error when the path can't be decoded